collector/transport_node_collector.go

```diff
@@ -28,7 +30,7 @@ func newTransportNodeCollector(apiClient *nsxt.APIClient, logger log.Logger) pro
 	transportNodeStatus := prometheus.NewDesc(
 		prometheus.BuildFQName(namespace, "transport_node", "status"),
 		"Status of Transport Node UP/DOWN",
-		[]string{"id", "name"},
+		[]string{"id", "name", "type", "edge_member_index"},
```

Comment:
As edge_member_index is specific to edge transport node, can we use transport_node_edge_membership instead? The labels are id and edge_member_index, this work like this article
https://www.robustperception.io/how-to-have-labels-for-machine-roles
I'm not sure membership can work the same way, but it's an interesting alternative

Reply:
fixed in 7c8248d